src/modules/editor/ScriptEditorImplementation.cpp

```diff
@@ -475,6 +496,7 @@ ScriptEditorWidgetColorOptions::ScriptEditorWidgetColorOptions(QWidget * pParent
 	gbox->setInsideSpacing(0);
 
 	addColorSelector(gbox, __tr2qs_ctx("Background:", "editor"), &g_clrBackground, true);
+	addColorSelector(gbox, __tr2qs_ctx("Background Highlight:", "editor"), &g_clrBackgroundHighlight, true);
```

Comment:
I'd agree with changing it to "Background highlight", but I'm not in favor of "Highlight current line" since I am still deciding if I want to add to it a multi-line highlighter when you do a multi-line selection. So, labeling it as "current line" wouldn't make sense.

Also, It's a replacement background color for when the cursor is on that line, so labeling it as "Background highlight" makes more sense than "Highlight current line" which seems like it could be several different things like "transparency setting" or something.

Maybe you'd prefer "Highlight background"?